lms/djangoapps/mobile_api/users/views.py

```diff
@@ -60,7 +60,7 @@ class UserDetail(generics.RetrieveAPIView):
     """
     queryset = (
         User.objects.all()
-        .select_related('profile', 'course_enrollments')
+        .select_related('profile')
```

Comment:
The actual course_enrollment was needed because DRF only builds a link to the object in the response JSON, using the username.